src/chrome/content/rules/Chefkoch.de.xml

```diff
@@ -0,0 +1,16 @@
+<ruleset name="Chefkoch (partial)">
+
+	<target host="*.chefkoch.de" />
+	<target host="*.chefkoch-cdn.de" />
+	<target host="*.chefkoch-blog.de" />
+
+	<rule from="^http://(css|emskr|img|js|static)\.chefkoch-cdn\.de/"
+		to="https://$1.chefkoch-cdn.de/" />
+		
+	<rule from="^http://cdn\.chefkoch\.de/"	
+		to="https://chefkoch.de/" />
```

Comment:
Did you mean?

`<rule from="^http://cdn\.chefkoch\.de/"  to="https://chefkoch.de/" />`